fiat payment (#1556)
w3f/Grants-Program: .github/pull_request_template.md

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [ ] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).

> - [ ] A BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address for the payment of the milestones is provided in the application.

Reviewer: ```suggestion
- [ ] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address in the application).
```
